testdata/project-v2-addon/config/crd/patches/webhook_in_admirals.yaml

```diff
@@ -8,9 +8,6 @@ spec:
   conversion:
     strategy: Webhook
     webhookClientConfig:
-      # this is "\n" used as a placeholder, otherwise it will be rejected by the apiserver for being blank,
-      # but we're going to set it later using the cert-manager (or potentially a patch if not using cert-manager)
-      caBundle: Cg==
```

Comment:
The change cannot be done for v2. Only for v3+. It would be a breaking change.

Reply:
Thanks for the reply @camilamacedo86.

Can you explain more why this is a breaking change for v2? The comment in question is wrong - this field (`caBundle`) is optional and as far as I can tell has been optional since at least 1.13 (possibly longer). The CRD will not be rejected if this field is omitted.

Is there some behavioral breakage we're concerned with here?

I'm not super familiar with the v2 vs v3 distinction in Kubebuilder so just trying to understand why one is ok and one isn't.

Reply:
Hi @matthchr, 

Just bugs we are addressing to v2+ ( pkg/plugin/v2/) which means that this should be done only in ` pkg/plugin/v3/`. Could you please revert the change in the files inside of ` pkg/plugin/v2/`?

Reply:
Did this